doc/manuals/devel/mongoBackend.md

```diff
@@ -168,117 +169,117 @@ Case 6: action type is "DELETE" to remove an entity
-* If at least some attribute has the `found` flag set to `false`, a lookup for matching registrations based on specific attributes (i.e. in the form "E-A") is done, calling the `registrationsQuery()` function in the `MongoGlobal` module (step 2). This function search in the DB using the `collectionRangedQuery()` in the `connectionOperations` module (steps 3 and 4).
-* Then, the `fillContextProviders()` function (in the `MongoGlobal` module) is called in order to attempt to fill the not found attributes with the matched registrations (step 5).
-* If at least some attribute stills with the `found` flag set to `false` a new lookup round is done. This time, searching for whole entities, (i.e. in the "E-&lt;null&gt;" form). Again, the `registrationsQuery()` function is used (step 6). This function search in the DB using the `collectionRangedQuery()` in the `connectionOperations` module (steps 7 and 8).
-* Then, the `fillContextProviders()` function (in the `MongoGlobal` module) is called again in order to attempt to fill the not found attributes with the new matched registrations (step 9).
+* `searchContextProviders()` is invoked (step 1) from one of four possible flows. See diagrams [MB-01](#flow-mb-01), [MB-02](#flow-mb-02), [MB-03](#flow-mb-03) and [MB-05](#flow-mb-05). Apart from these entry points, note that `searchContextProviders()` can also be called from `updateEntity()`, in case `processContextAttributeVector()` fails (which means that the entity wasn't actually modified locally, so it makes sense to search for Context Providers).
+* If at least one attribute has the `found` flag set to `false`, a lookup for matching registrations based on specific attributes (i.e. in the form "E-A") is done, calling `registrationsQuery()` in the `MongoGlobal` module (step 2). This function searches the database using `collectionRangedQuery()` in the `connectionOperations` module (steps 3 and 4).
+* Then, `fillContextProviders()` (in the `MongoGlobal` module) is called to attempt to fill the not found attributes with the matching registrations (step 5).
+* If at least one attribute still has the `found` flag set to `false`, a new lookup round is done. This time, searching for whole entities (i.e. in the "E-&lt;null&gt;" form). Again, `registrationsQuery()` is used (step 6). This function searches the database using `collectionRangedQuery()` in the `connectionOperations` module (steps 7 and 8).
+* Then, `fillContextProviders()` (in the `MongoGlobal` module) is called again to attempt to fill the not found attributes with the new matched registrations (step 9).
 
 [Top](#top)
 
 #### `mongoQueryContext` (SR)
 
-Encapsulates the logic for the query context operation.
+`mongoQueryContext()` encapsulates the logic for the query context operation.

```

Comment:
I think that here it refers to the module name, not the function. Thus, `()` shouldn't be used.

Maybe are other case of this "anti-style"? Plase, review it globaly in mongoBackend.md